hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/checksum/TestContainerChecksumTreeManager.java

```diff
@@ -59,6 +66,19 @@ class TestContainerChecksumTreeManager {
   private ContainerMerkleTreeMetrics metrics;
   private ConfigurationSource config;
 
+  public static Stream<Arguments> getParameters() {
+    return Stream.of(
+        Arguments.of(1, 2, 3),
+        Arguments.of(2, 3, 1),
+        Arguments.of(3, 1, 2),
+        Arguments.of(2, 2, 3),
+        Arguments.of(3, 2, 2),
+        Arguments.of(3, 3, 3),
+        Arguments.of(2, 1, 4),
+        Arguments.of(2, 3, 4),
+        Arguments.of(1, 2, 4));
```

Comment:
Broadly speaking the three cases to test are zero, one, or many. Zero and one are useful to isolate problems before proceeding to more complex tests, so we should definitely have these in the matrix:
```
(0, 0, 0)
(0, 0, 1)
(0, 1, 0)
(1, 0, 0)
```
Next we want to test combinations of faults. We want to make sure that one type of detection does not depend on another, so the matrix might have these entries as well:
```
(3, 3, 3)
(3, 3, 0)
(3, 0, 3)
(0, 3, 3)
```
I think this will provide decent coverage, but it might need to be updated when we add deleted block testing.